### PR TITLE
Add Event Created

### PR DESCRIPTION
**Pull request by serotonie**
Hi there,

I needed to take some action when an event is created, so I added this webhook, if it interests you.

Cheers for the initial work ! :)

### CODE REVIEW

File: lib/Listeners/CalendarObjectCreatedListener.php [hunk 1]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (c) 2021 Paweł Kuffel <pawel@kuffel.io>
+ *
+ * @author Paweł Kuffel <pawel@kuffel.io>
+ *
+ * @license GNU AGPL version 3 or any later version
```
Comment (kffl) [inline lib/Listeners/CalendarObjectCreatedListener.php:8]: Feel free to modify the copyright notice so as to list yourself as the author in this file.

**Review by kffl — COMMENTED** [review]
LGTM

**Comment by serotonie** [comment]
> Hi @[serotonie](https://github.com/serotonie), thank you for taking the time to create this PR.
> 
> Would you mind documenting the newly added event type in README? If not, no worries, I can do that before releasing a new version.

Hi @kffl,
Yeah will do it ASAP. My weekend shouldn't be too busy, I should be able to do it till Sunday evening (UTC+2)

**Comment by serotonie** [comment]
Documentation is up to date

**Comment by kffl** [comment]
Thanks a lot! Your changes will be included in v0.3.0 that I will release in a second.

**Pull request merged**